tardis/io/util.py

```diff
@@ -166,7 +166,91 @@ def check_equality(item1, item2):
         return True
 
 
-def to_hdf(path_or_buf, path, elements, complevel=9, complib='blosc'):
+class HDFReaderWriter():
+
+    @classmethod
+    def to_hdf(cls, path_or_buf, path, elements, complevel=9, complib='blosc'):
+        """
+        A function to uniformly store TARDIS data
+        to an HDF file.
+
+        Scalars will be stored in a Series under path/scalars
+        1D arrays will be stored under path/property_name as distinct Series
+        2D arrays will be stored under path/property_name as distinct DataFrames
+
+        Units will be stored as their CGS value
+
+        Parameters
+        ----------
+        path_or_buf:
+            Path or buffer to the HDF store
+        path: str
+            Path inside the HDF store to store the `elements`
+        elements: dict
+            A dict of property names and their values to be
+            stored.
+
+        Returns
+        -------
+
+        """
+        scalars = {}
+        for key, value in elements.iteritems():
+            if hasattr(value, 'cgs'):
+                value = value.cgs.value
+            if np.isscalar(value):
+                scalars[key] = value
+            elif hasattr(value, 'shape'):
+                if value.ndim == 1:
+                    # This try,except block is only for model.plasma.levels
```

Comment:
you suggest that this is only relevant for `model.plasma.levels` then maybe it should live there as an addition to the `to_hdf` in this specific class.  But we can discuss this later.